src/include/rats-tls/api.h

```diff
@@ -38,12 +38,7 @@ typedef enum {
 	RATS_TLS_CERT_ALGO_RSA_3072_SHA256,
 	RATS_TLS_CERT_ALGO_ECC_256_SHA256,
 	RATS_TLS_CERT_ALGO_MAX,
-/* FIXME: need to look into why openssl 1.0 cannot work with ECC-256 */
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
 	RATS_TLS_CERT_ALGO_DEFAULT = RATS_TLS_CERT_ALGO_RSA_3072_SHA256
```

Comment:
I don't think this is a solution. Using ECC can speed up the performance of cert generation.

Reply:
yes, but ECC is not suitable for the default signing algorithm.  It can be an option for users who can understand the ECC-256 and prefer to use ECC explicitly.

Reply:
I think the root cause is this issue is not resolved and introduces regression https://github.com/inclavare-containers/inclavare-containers/issues/1138

In addition, I don't understand why openssl 1.0.x cannot handle ECC-256. It doesn't support the generation of ECC-256 cert or verify the input ECC-256 cert? As I know, ECC-256 is supported by openssl 1.0.x perfectly. Or intel-sgx-ssl doesn't support ECC-256 without re-configuration during the build?

Reply:
Another related issue: https://github.com/inclavare-containers/inclavare-containers/issues/1261 @YangLiang3 I think we need to have a root cause analysis.